src/hammer-vlsi/defaults.yml

```diff
@@ -316,14 +316,15 @@ vlsi.inputs:
     y: 0
     # pitch (float) - pitch of bumps in microns
     pitch: 0.0
-    # cell (str) - Name of bump cell
+    # cell (str) - Name of the default bump cell
     cell: ""
     # assignments - List of BumpAssignment structs. You must specify one of name or no_connect.
     # If both are specified the bump will be left unconnected
     #  - name (Optional[str]) - The name of the net being assigned to a bump
     #  - no_connect (Optional[bool]) - If True the bump will be present but unconnected
-    #  - x (int) - The X coordinate of the bump assigned to this net
-    #  - y (int) - The Y coordinate of the bump assigned to this net
+    #  - x (float) - The X coordinate of the bump assigned to this net
+    #  - y (float) - The Y coordinate of the bump assigned to this net
```

Comment:
`floats` are dangerous here. `Decimal` would be better.

Reply:
(see #376)

Reply:
type hint changed to Decimal.